diff --git a/dist/IconProps.d.ts b/dist/IconProps.d.ts
index f0658817539059bd51326737390a590b4216ce81..abab920ec8d15c809f98131817e1f7081f8daf33 100644
--- a/dist/IconProps.d.ts
+++ b/dist/IconProps.d.ts
@@ -1,5 +1,6 @@
-import { CSSProperties, RefObject, AriaAttributes } from 'react';
-export interface HTMLProps extends AriaAttributes {
+import { CSSProperties, RefObject } from 'react';
+export interface HTMLProps {
+    [x: `aria-${string}`]: string | boolean | number | null;
     className?: string;
 }
 export interface IconProps extends HTMLProps {
diff --git a/dist/StackProps.d.ts b/dist/StackProps.d.ts
index 7ee96421f7e1ac8344f4c9bd04ea2d153d9c904b..95590c9a210f3850c1ec6ece15e3079cfd619022 100644
--- a/dist/StackProps.d.ts
+++ b/dist/StackProps.d.ts
@@ -1,5 +1,5 @@
-import { ReactElement, CSSProperties, RefObject } from "react";
-import { IconProps, HTMLProps } from "./IconProps";
+import { CSSProperties, RefObject } from "react";
+import { HTMLProps } from "./IconProps";
 export interface StackPropsBase {
     ref?: RefObject<SVGSVGElement>;
     title?: string | null;
@@ -13,5 +13,5 @@ export interface StackPropsBase {
     style?: CSSProperties;
 }
 export interface StackProps extends StackPropsBase, HTMLProps {
-    children: ReactElement<IconProps>[] | ReactElement<IconProps>;
+    children: any;
 }
