diff --git a/index.d.mts b/index.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..b4fa6c1cb6f65aab31e2a87a3cec87ddd3e8dd6b
--- /dev/null
+++ b/index.d.mts
@@ -0,0 +1,2 @@
+export * from 'preact/compat';
+export {default} from 'preact/compat';
diff --git a/index.d.ts b/index.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..8f5c094616ce0c028dd36ff0c00e7e5b61915137
--- /dev/null
+++ b/index.d.ts
@@ -0,0 +1,2 @@
+import * as mod from 'preact/compat';
+export = mod;
diff --git a/jsx-dev-runtime.d.mts b/jsx-dev-runtime.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..528b906cc677216054d6676dfc792a46c0ae1b1e
--- /dev/null
+++ b/jsx-dev-runtime.d.mts
@@ -0,0 +1,2 @@
+export * from 'preact/jsx-runtime';
+export {default} from 'preact/jsx-runtime';
diff --git a/jsx-dev-runtime.d.ts b/jsx-dev-runtime.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..be1ed1ffdd73870ecf78ed121b81f59614589cf7
--- /dev/null
+++ b/jsx-dev-runtime.d.ts
@@ -0,0 +1,2 @@
+import * as mod from 'preact/jsx-runtime';
+export = mod;
diff --git a/jsx-runtime.d.mts b/jsx-runtime.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..528b906cc677216054d6676dfc792a46c0ae1b1e
--- /dev/null
+++ b/jsx-runtime.d.mts
@@ -0,0 +1,2 @@
+export * from 'preact/jsx-runtime';
+export {default} from 'preact/jsx-runtime';
diff --git a/jsx-runtime.d.ts b/jsx-runtime.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..be1ed1ffdd73870ecf78ed121b81f59614589cf7
--- /dev/null
+++ b/jsx-runtime.d.ts
@@ -0,0 +1,2 @@
+import * as mod from 'preact/jsx-runtime';
+export = mod;
diff --git a/server.d.mts b/server.d.mts
new file mode 100644
index 0000000000000000000000000000000000000000..a4475cdd0381d1cb6a299b692bb4a33c282dccc4
--- /dev/null
+++ b/server.d.mts
@@ -0,0 +1,2 @@
+export * from 'preact/compat/server';
+export {default} from 'preact/compat/server';
diff --git a/server.d.ts b/server.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..ffbf7bbc39449d83db80f5b18ea8cba3029dfa25
--- /dev/null
+++ b/server.d.ts
@@ -0,0 +1,2 @@
+import * as mod from 'preact/compat/server';
+export = mod;